app/components/pages/AuthorGuide/Fees.js

```diff
@@ -0,0 +1,275 @@
+import React from 'react'
+import { H1, H2, H3, Link } from '@pubsweet/ui'
+
+import Paragraph from '../../ui/atoms/Paragraph'
+import ExternalLink from '../../ui/atoms/ExternalLink'
+import List from '../../ui/atoms/List'
+
+const Fees = props => (
+  <React.Fragment>
+    <H1>Publication Fees FAQs</H1>
+
+    <Paragraph>
+      A fee of $2,500* is collected for published papers; however, authors with
+      insufficient funding to pay the fee are eligible for a fee waiver.
+    </Paragraph>
+
+    <Paragraph>
+      The publication fee (also referred to as an “open-access” or
+      &quot;article-processing&quot; fee) is a charge collected by many
```

Comment:
In the original, both `open-access` and `article-processing` are surrounded by curly quotes but here the latter has straight quotes.